docs/topics/native/native-target-support.md

```diff
@@ -0,0 +1,100 @@
+| Apple macOS hosts only  |                                   |               |                                                    |
+| `watchosSimulatorArm64` | `aarch64-apple-watchos-simulator` | ✅             | Apple watchOS simulator on Apple Silicon platforms |
+| `watchosX64`            | `x86_64-apple-watchos-simulator`  | ✅             | Apple watchOS 64-bit simulator on x86_64 platforms |
+| `watchosArm32`          | `armv7k-apple-watchos`            |               | Apple watchOS on ARM32 platforms                   |
+| `watchosArm64`          | `arm64_32-apple-watchos`          |               | Apple watchOS on ARM64 platforms with ILP32        |
+| `tvosSimulatorArm64`    | `aarch64-apple-tvos-simulator`    | ✅             | Apple tvOS simulator on Apple Silicon platforms    |
+| `tvosX64`               | `x86_64-apple-tvos-simulator`     | ✅             | Apple tvOS simulator on x86_64 platforms           |
+| `tvosArm64`             | `aarch64-apple-tvos`              |               | Apple tvOS on ARM64 platforms                      |
+| `iosArm64`              | `aarch64-apple-ios`               |               | Apple iOS and iPadOS on ARM64 platforms            |
+
+> We're doing our best to move `iosArm64` to Tier 1, as it's a crucial target for [Kotlin Multiplatform Mobile](multiplatform-mobile-getting-started.md).
+> To do that, we need first to create a dedicated testing infrastructure because platform limitations make it difficult
+> to run compiler tests on Apple devices.
+> 
+> Meanwhile, we sometimes run tests manually on iOS devices and rely on testing
+> similar targets, like `iosSimulatorArm64`, which should be sufficient in most cases.
+> 
+{type="tip"}
+
+## Tier 3
+
+* The target is not guaranteed to be tested on CI.
+* We can't promise a source and binary compatibility between different compiler releases, though such changes for these
+  targets are quite rare.
+
+| Gradle target name     | Target triple                   | Running tests | Description                                                          |
+|------------------------|---------------------------------|---------------|----------------------------------------------------------------------|
+| `androidNativeArm32`   | `arm-unknown-linux-androideabi` |               | [Android NDK](https://developer.android.com/ndk) on ARM32 platforms  |
+| `androidNativeArm64`   | `aarch64-unknown-linux-android` |               | [Android NDK](https://developer.android.com/ndk) on ARM64 platforms  |
+| `androidNativeX86`     | `i686-unknown-linux-android`    |               | [Android NDK](https://developer.android.com/ndk) on x86 platforms    |
+| `androidNativeX64`     | `x86_64-unknown-linux-android`  |               | [Android NDK](https://developer.android.com/ndk) on x86_64 platforms |
+| `mingwX64`             | `x86_64-pc-windows-gnu`         | ✅             | 64-bit Microsoft Windows                                             |
```

Comment:
It is not a "real" Windows target (which is MSVC), but a MinGW, which is, roughly speaking, a way to port Linux apps to Windows. It has a different API and ABI, a different set of quirks, so users should know what they use.
So a better description would be something like this:
> 64-bit [MinGW](https://www.mingw-w64.org/) (Windows 7 and newer)